File: Python Word Examples/DigitalSignatureApiDemo/addDigitalSignature.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.digital_signature_api.DigitalSignatureApi(configuration)

file_name = 'AddDigitalSignature.docx'
pfx_file = 'D:/input/Gary.pfx'
secure_password = 'e-iceblue'
dest_file_path = 'digitalSignature/AddDigitalSignature.docx'
extra = {'folder': 'input'}
api.add_digital_signature('Template.docx', pfx_file, secure_password, dest_file_path, **extra)

File: JavaScript Word Examples/DigitalSignatureApiDemo/addDigitalSignature.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword= require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.DigitalSignatureApi(apiClient);

    var filename = "Template.docx";
    var fs = require('fs');
    var pfxFile = fs.createReadStream("D:/gary.pfx");
    var pfxFilePassword = "e-iceblue";
    var destFilePath = 'output/AddDigitalSignature.docx';
    var opt = { folder: 'input' };
    var response = instance.addDigitalSignature(filename, pfxFile, pfxFilePassword, destFilePath, opt, 
        function (error, data, response) {
            if (error) throw error;
        });
})();